### PR TITLE
CDP-360 Include DSR dashboard under save route

### PR DESCRIPTION
**Pull request by sirromdev**
Should also satisfy CDP-362 && CDP-361

### CODE REVIEW

File: src/components/AccountBox.js [hunk 1]
```diff
@@ -1,31 +1,129 @@
...
+              t="p5"
+              textAlign="left"
+              width="30%"
+            >
+              {balanceUSD}
+            </Text>
+            {btn}
+          </Flex>
+        ))}
+      </CardBody>
```
Comment (sirromdev) [inline src/components/AccountBox.js]: Stubbed layout of Wallet balances. Can fill in later with live data when we have more designs.

File: src/components/ActiveAccount.js [hunk 2]
```diff
@@ -1,38 +1,45 @@
...
       <Box ml="s">
-        <Text t={addressTextStyle} color={textColor}>
-          {cutMiddle(address, 7, 5)}
-        </Text>
+        {address ? (
+          <Text t={addressTextStyle} color={textColor}>
+            {cutMiddle(address, 7, 5)}
+          </Text>
+        ) : null}
       </Box>
```
Comment (sirromdev) [inline src/components/ActiveAccount.js]: These are the updates to the sidebar wallet dropdown. Is it necessary to hide this component when on `/borrow` as we would, in that case, have two wallet logins?

File: src/components/CDPList.js [hunk 3]
```diff
@@ -93,9 +95,10 @@ const CDPList = memo(function({ currentPath, viewedAddress, currentQuery }) {
           />
         );
       })}
-      {!account ? null : (
+      {!showPlusButton ? null : (
         <DashedFakeButton
           onClick={() =>
+            account &&
             show({ modalType: 'cdpcreate', modalTemplate: 'fullscreen' })
```
Comment (sirromdev) [inline src/components/CDPList.js]: The design shows a + button to create a cdp, even when on `/borrow`. Not that important but we may want to retract this

File: src/components/Navbar.js [hunk 4]
```diff
@@ -11,10 +11,14 @@ import { Routes } from '../utils/constants';
 const Navbar = ({ viewedAddress }) => {
   const { url } = useCurrentRoute();
   const { account } = useMaker();
+  const onBorrowLandingPage = url.pathname === `/${Routes.BORROW}`;
   const onOverviewPage =
     account && url.pathname === `/${Routes.BORROW}/owner/${account.address}`;
   return (
-    <Box bg={account ? 'blackLight' : 'white'} height="100%">
+    <Box
+      bg={account || onBorrowLandingPage ? 'blackLight' : 'white'}
```
Comment (sirromdev) [inline src/components/Navbar.js]: This change means that we would only show the portal as `readOnly` when having no account and under the overview or cdp page

File: src/layouts/AppLayout.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+        <Subheader />
+        {children}
+        <Box flexGrow="1" />
+        <Footer />
+      </Flex>
+    </Grid>
+  );
+};
+
+export default hot(AppLayout);
```
Comment (sirromdev) [inline src/layouts/AppLayout.js]: This is the layout for the Savings dashboard.

File: src/pages/Borrow.js [hunk 6]
```diff
@@ -111,4 +111,4 @@ function Auth() {
   );
 }
 
-export default hot(Auth);
+export default hot(Borrow);
```
Comment (sirromdev) [inline src/pages/Borrow.js]: Renaming this file made sense to me, `Auth` doesn't describe it quite so well now that we have included Savings

File: src/pages/Landing.js [hunk 7]
```diff
@@ -2,31 +2,48 @@ import React from 'react';
...
+          <Flex flexDirection="column">
+            <Title pb="l" textAlign="center">
+              Oasis
+            </Title>
+            <Flex pb="l" flexDirection="row" justifyContent="space-around">
+              <CardLink title="Borrow" link={Routes.BORROW} />
+              <CardLink title="Save" link={Routes.SAVE} />
+            </Flex>
+          </Flex>
         </LandingHeroLayout>
```
Comment (sirromdev) [inline src/pages/Landing.js]: Rough outline for the Oasis Landing page. Just provides two Card links for the relevant sections

**Comment by sirromdev** [comment]
The above is a current issue with horizontal overflow on the sidebars cannot replicate on my machine.

File: src/components/ActiveAccount.js (hunk 2 above)
Comment (levity) [reply src/components/ActiveAccount.js]: The Save pages are going to be revised by the design team so they make sense when integrated with the CDP portal. Probably we'll show this component again in an appropriate place. For now, just remove the placeholder account box that shows up in the upper right of the Save pages.

File: src/components/CDPList.js (hunk 3 above)
Comment (levity) [reply src/components/CDPList.js]: Good point--better to hide the button entirely

**Comment by levity** [comment]
Note that CDP-362 is about actually implementing the functionality using the SavingsService from dai-plugin-mcd. It looks like you should hold off on that until at least after the next planning/grooming session.

**Pull request merged**